lib/YuiRestClient/Wait.pm

```diff
@@ -12,10 +12,12 @@ sub wait_until {
     $args{message} //= '';
 
     die "No object passed to the method" unless $args{object};
-
-    my $counter = abs(int($args{timeout} / $args{interval}));
     my $result;
-    while (--$counter >= 0) {
+    # if timeout is 0, just execute the passed expression once
+    return eval { $args{object}->() } if $args{timeout} == 0;
```

Comment:
do we need in our framework timeout=0, perhaps it is more obvious for the user to have an error which say "don't use timeout 0, instead use just the function as it is'. I don't know if you borrow that concept from openQA or other framework, might not make sense, and we will just have one `eval` in the function. wdyt?